catalogue/webapp/pages/work.js

```diff
@@ -210,6 +199,7 @@ WorkPage.getInitialProps = async (
 
   const { id, query, page } = ctx.query;
   const workOrError = await getWork({ id });
+  const { showSingleImageWorkPreview } = ctx.query.toggles;
```

Comment:
FYI - To be typesafe, (Need to get flow on this), having  `const { showSingleImageWorkPreview = false } = ctx.query.toggles;` ensures a Boolean. Not that it matters because JS...

Reply:
This is dealt with in another branch, branched off this one